pkg/pgmodel/querier/query_builder.go

```diff
@@ -6,11 +6,13 @@ package querier
 
 import (
 	"fmt"
-	"github.com/timescale/promscale/pkg/log"
 	"math"
+	"regexp"
```

Comment:
We no longer use regex across the code. So, I think this should be

```suggestion
	github.com/grafana/regex
```